.gitignore

```diff
@@ -9,22 +9,12 @@ tracking/__pycache__/
 
 tracking/save/
 
-event_tracking/save/
-gimbal_lock/save/
-models/wu_converted_definitif_without_floating_base_template_with_variables.bioMod
+kinova_tracking/inverse_kinematics_run.py
 
-models/wu_converted_definitif_without_floating_base_template_fixed_with_variables.bioMod
+kinova_tracking/inverse_kinematics.py
 
-models/wu_converted_definitif_without_floating_base_template_quat_with_variables.bioMod
-
-tracking/animate_my_solution.py
-
-Animate_my_model.py
+kinova_tracking/__pycache__/
 
 models/__pycache__/
 
-event_tracking/ocp_graph.gv.pdf
-
-event_tracking/ocp_graph.gv
-
-data/__pycache__/
```

Comment:
Dont remove everything.
Keep the line that should be kept.

Reply:
I don't modified the git ignore